models/articles.models.js

```diff
@@ -35,4 +35,21 @@ exports.fetchCommentsForArticle = (article) => {
     return db.query(queryStr,[id]).then(({rows}) => {
         return rows;
     })
-}   
\ No newline at end of file
+}  
+
+exports.addCommentOnArticle = (article, comment) => {
+
+    if( typeof comment.username !== 'string' || typeof comment.body !== 'string'){
```

Comment:
this is not necessary as psql would throw an error if the username is invalid or if the body is not a string